### PR TITLE
Feature coverage

### PR DESCRIPTION
**Pull request by WallyMaier**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: regression/scripts/Vehicles/BWB.py [hunk 1]
```diff
@@ -0,0 +1,402 @@
+# BWB.py
```
Comment (planes) [inline regression/scripts/Vehicles/BWB.py]: simplify out this script please

**Comment by planes** [comment]
This is displaying a little better on my end.

File: regression/scripts/Vehicles/BWB.py (hunk 1 above)
Comment (WallyMaier) [reply regression/scripts/Vehicles/BWB.py]: which parts?  it is consistent with other vehicle scripts.

Comment (planes) [reply regression/scripts/Vehicles/BWB.py]: I would like this to not be able to be executed and run through a mission. I don't want to have to support and aircraft of questionable validity. Does it need a turbofan or multiple configs?

Comment (WallyMaier) [reply regression/scripts/Vehicles/BWB.py]: done

**Pull request merged**